Automattic/jetpack
# Status: Decouple `is_woa_site` from `at_options`

#### Changes proposed in this Pull Request:

The `at_options` option was only used by WoA sites to store the slug of the plan currently active on the site. However, as we shifted towards storing purchases in the Atomic persistent data, the `at_options` option is no longer a reliable source to determine whether a site is a WoA site.

To prevent unexpected side effects, this PR removes the `at_options` check from the `is_woa_site`, so it only checks now the presence of the `wpcomsh` plugin.


#### Other information:


#### Jetpack product discussion

pdtR3Z-12U-p2 / D82683-code

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Apply these changes to a WoA site
- Make sure that `is_woa_site` still returns true
